drivers/input/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ zephyr_library_sources_ifdef(CONFIG_INPUT_GPIO_QDEC input_gpio_qdec.c)
 zephyr_library_sources_ifdef(CONFIG_INPUT_GT911 input_gt911.c)
 zephyr_library_sources_ifdef(CONFIG_INPUT_NPCX_KBD input_npcx_kbd.c)
 zephyr_library_sources_ifdef(CONFIG_INPUT_XPT2046 input_xpt2046.c)
+zephyr_library_sources_ifdef(CONFIG_INPUT_STMPE811 input_stmpe811.c)
```

Comment:
can you keep these block aphabetically sorted?

Reply:
This is now changed